src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,204 @@
+package duke;
+
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.io.File;
+
+import duke.exception.IllegalCommandException;
+import duke.exception.EmptyCommandException;
+import duke.task.Task;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Todo;
+
+public class Duke {
+
+    public static ArrayList<Task> tasks = new ArrayList<>();
+    public static final String FILE_NAME = "duke.txt";
+    public static final String DIR_NAME = "data";
+    public static final String FILE_PATH = DIR_NAME + "/" + FILE_NAME;
+
+    /**
+     * Prints a farewell message when the user exits the program.
+     */
+    public static void commandBye() {
```

Comment:
Name of this method sounds like a noun, but since it is printing something, it should be a verb eg. `printByeCommand`